tests/unit/CMakeLists.txt

```diff
@@ -282,42 +278,67 @@ target_link_libraries(${test_prefix}commit_log_v2 gflags mg-utils mg-storage-v2)
 add_unit_test(property_value_v2.cpp)
 target_link_libraries(${test_prefix}property_value_v2 mg-storage-v2 mg-utils)
 
-add_unit_test(storage_v2.cpp)
-target_link_libraries(${test_prefix}storage_v2 mg-storage-v2 storage_test_utils)
-
+# add_unit_test(storage_v2.cpp)
+# target_link_libraries(${test_prefix}storage_v2 mg-storage-v2 storage_test_utils)
 add_unit_test(storage_v2_constraints.cpp)
 target_link_libraries(${test_prefix}storage_v2_constraints mg-storage-v2)
 
 add_unit_test(storage_v2_decoder_encoder.cpp)
 target_link_libraries(${test_prefix}storage_v2_decoder_encoder mg-storage-v2)
 
-add_unit_test(storage_v2_durability.cpp)
-target_link_libraries(${test_prefix}storage_v2_durability mg-storage-v2)
-
-add_unit_test(storage_v2_edge.cpp)
-target_link_libraries(${test_prefix}storage_v2_edge mg-storage-v2)
+# add_unit_test(storage_v2_durability.cpp)
+# target_link_libraries(${test_prefix}storage_v2_durability mg-storage-v2)
 
+# add_unit_test(storage_v2_edge.cpp)
+# target_link_libraries(${test_prefix}storage_v2_edge mg-storage-v2)
 add_unit_test(storage_v2_gc.cpp)
 target_link_libraries(${test_prefix}storage_v2_gc mg-storage-v2)
 
-add_unit_test(storage_v2_indices.cpp)
-target_link_libraries(${test_prefix}storage_v2_indices mg-storage-v2 mg-utils)
-
+# add_unit_test(storage_v2_indices.cpp)
+# target_link_libraries(${test_prefix}storage_v2_indices mg-storage-v2 mg-utils)
 add_unit_test(storage_v2_name_id_mapper.cpp)
 target_link_libraries(${test_prefix}storage_v2_name_id_mapper mg-storage-v2)
 
 add_unit_test(storage_v2_property_store.cpp)
 target_link_libraries(${test_prefix}storage_v2_property_store mg-storage-v2 fmt)
 
-add_unit_test(storage_v2_wal_file.cpp)
-target_link_libraries(${test_prefix}storage_v2_wal_file mg-storage-v2 fmt)
-
-add_unit_test(storage_v2_replication.cpp)
-target_link_libraries(${test_prefix}storage_v2_replication mg-storage-v2 fmt)
+# add_unit_test(storage_v2_wal_file.cpp)
+# target_link_libraries(${test_prefix}storage_v2_wal_file mg-storage-v2 fmt)
 
+# add_unit_test(storage_v2_replication.cpp)
+# target_link_libraries(${test_prefix}storage_v2_replication mg-storage-v2 fmt)
```

Comment:
does it make more sense to remove these, rather than have them commented out?

Reply:
Hmm I don't see why, unless we want to explicitly track for some time which tests we have moved and which we haven't

Reply:
What do you say @antaljanosbenjamin?

Reply:
> unless we want to explicitly track for some time which tests we have moved and which we haven't

How does this connected to this? I don't see it.

In general I think we should barely touch things that are on master, because we will constantly merging master to project pineapples. So maybe it make sense revert all the changes in v2 to leave it untouched.

Reply:
But we don't seperate tests, and we agreed that the new tests would have prefix storage_v3 or query_v2, but we would keep the old tests as well, the issues is that we cannot leave them running since they will be always failing. So we can either comment them out and always take a quick look to see which ones we had to convert or just delete them, is that clear?

Reply:
This will be handled in the next PR which moves all the changes do appropriate versions of storage and query engine